src/main/java/com/feywild/feywild/block/entity/FeyAltar.java

```diff
@@ -92,8 +89,7 @@ public void tick() {
                     for (int slot = 0; slot < this.inventory.getSlots(); slot++) {
                         this.inventory.setStackInSlot(slot, ItemStack.EMPTY);
                     }
-                    ItemEntity entity = new ItemEntity(this.level, this.worldPosition.getX() + 0.5, this.worldPosition.getY() + 2, this.worldPosition.getZ() + 0.5, currentRecipe.getLeft().copy());
-                    this.level.addFreshEntity(entity);
+
```

Comment:
Why was this code removed?

```java
ItemEntity entity = new ItemEntity(this.level, this.worldPosition.getX() + 0.5, this.worldPosition.getY() + 2, this.worldPosition.getZ() + 0.5, currentRecipe.getLeft().copy());
this.level.addFreshEntity(entity);
```

Reply:
Restored this, dont know why that happened. I check it as example for adding an item.